build/commands/lib/config.js

```diff
@@ -1205,8 +1226,23 @@ Object.defineProperty(Config.prototype, 'defaultOptions', {
 
       // Upload stats about Goma actions to the Goma backend.
       env.GOMA_PROVIDE_INFO = true
+    }
+
+    if (this.rbeService) {
```

Comment:
what happens if both goma and rbe are set? Maybe as above we should list rbe first and if/else?

Reply:
> what happens if both goma and rbe are set? Maybe as above we should list rbe first and if/else?

`gn gen` will fail if both goma and remoteexec are enabled. We pass both gn args related to goma and rbe enabling unconditionally.